dev/run-tests.py

```diff
@@ -477,7 +477,15 @@ def determine_java_version(java_exe):
 
     raw_output = subprocess.check_output([java_exe, "-version"],
                                          stderr=subprocess.STDOUT)
-    raw_version_str = raw_output.split('\n')[0]  # eg 'java version "1.8.0_25"'
+
```

Comment:
Is it maybe better still to look for the first line starting with "java version" instead? is that not maybe more explicit?